ethereum/referencetests/src/reference-test/java/org/hyperledger/besu/ethereum/vm/GeneralStateReferenceTestTools.java

```diff
@@ -153,8 +153,7 @@ public static void executeTest(final GeneralStateTestCaseEipSpec spec) {
             .blobGasPricePerGas(blockHeader.getExcessBlobGas().orElse(BlobGas.ZERO));
     final TransactionProcessingResult result =
         processor.processTransaction(
-            blockchain,
-            worldStateUpdater,
+                worldStateUpdater,
```

Comment:
this formatting seems out of line but I'm assuming spotless has it under control